source/fundamentals/crud/write-operations/change-a-document.txt

```diff
@@ -59,6 +59,15 @@ and their usage </reference/operator/update-field/>`.
 The update operators apply only to the fields associated with them in your
 update document.
 
+.. note:: Aggregation Pipelines in Update Operations
+
+   If you are using MongoDB Version 4.2 or later, you can use aggregation
+   pipelines made up of a subset of aggregation stages in update operations. For
+   more information, including what aggregation stages MongoDB supports in
```

Comment:
```suggestion
   more information on the aggregation stages MongoDB supports in
```
Suggestion: There doesn't seem to be any other information you can be referencing, so saying "including" was confusing for me.

Reply:
Thanks for the note. I agree that ", including what" can be replaced by "on the".  
Committing suggested change. 

I agree that "for more information, including..." is not very helpful (more information be be clarified right after it is mentioned).